charts/logzio-tracing/values.yaml

```diff
@@ -0,0 +1,28 @@
+Secrets:
```

Comment:
I don't work for logzio so don't pay me any attention.

I tried this helm chart and it worked. My only nitpick is that these configurations are capitalized, which is a departure with other logzio charts.